docker/bin/docker-entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
```

Comment:
Perhaps `set -e` ?

Reply:
better safe than sorry!

Reply:
> FYI https://github.com/NuSkooler/enigma-bbs/actions/runs/1766553445

this fails in     tags: enigmabbs/enigma-bbs:latest

due to missing python3, my code should fix it

Reply:
![image](https://user-images.githubusercontent.com/537154/151674361-0214248e-4a9f-4cc6-865e-02c2ae71d5bc.png)
![image](https://user-images.githubusercontent.com/537154/151674371-bef9dfa7-7409-4360-a124-4af20d34c287.png)